components/responsive images/responsive-images.css

```diff
@@ -0,0 +1,50 @@
+
+
+.circle-landscape-img,
+.circle-portrait-img,
+.circle-img {
+    background-color: red;
+    width: 10rem;
+    height: 10rem;
+    border-radius: 50%;
+}
+
+
+.circle-landscape-img,
+.circle-portrait-img {
+    position: relative;
+    display: inline-block;
+    overflow: hidden;
+}
+
+.circle-landscape-img img {
+    height: 100%;
+    width: auto;
+    transform: translateX(-35%);  
+    /* -ve translateX moves the image left relative to the container (i.e. horizontal direction), 
+    thus the container (and visible round image) relatively moves to the right inside the container */
+}
+
+.circle-portrait-img img{
+    height: auto;
+    width: 100%;
+    transform: translateY(-50%);
+    /* -ve translateY moves the image upwards relative to the container (i.e. vertical direction), 
+    thus the container (and visible round image) relatively moves downwards inside the container */
+}
+
+
+.fluid-img img {
+    width: 100%;
+    height: auto;
+}
+
+.fluid-img {
+    width: 80%;
+    height: auto;
```

Comment:
why are you repeating height: auto.

Reply:
.fluid-img selector is for the container div, and 
.fluid-img img selector is for the image element inside .fluid-img div